### ISSUE TITLE
Getting started - detailed guide for demo

### ISSUE THREAD

**Issue by chattarajoy** [bitdriftlabs/pulse#5]
It would be good to add a list of steps using which one can install the app and see it's capabilities. Here's a list of items that might be useful for users to see how things work:
* A helm chart
* Installation commands
* A default config that can read and write to certain prometheus endpoints along with some aggregations
* Steps: Access the admin endpoint to view the aggregations etc being done

In short: add documentation to make the crucial features easier to demo.
